frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts

```diff
@@ -43,6 +44,7 @@ describe('git import flow', () => {
     expect(importFromGitHeader.getText()).toContain('Import from git');
     await enterGitRepoUrl('https://github.com/sclorg/nodejs-ex.git');
 
+    scrollIntoView(appName);
```

Comment:
Interesting... I wouldn't think it needs to scroll to get elements... was this added reactively (you saw an error) or proactively (you are just being safe)?

Reply:
Since the submit button is sticky now, it overlaps with the input field and throws error while selecting it, so scrolling into view will avoid the overalap of the submit button.

Reply:
Oh... most interesting. I wonder if there are other issues similar to this... how does the form submit being off the page? Does it scroll down to the bottom in the tests (before your change)? Curious how the form submitted before, because the submit was always beyond "the fold".

Not sure I understand the issue here fully -- I may need to investigate more. Perhaps there is no "fold" in the tests... but due to the sticky footer, somehow it got in the way? Very confusing.

Reply:
Issue is sticky footer got in the way and  Webdriver is not able to select the input alone when the other buttons are overlapping.
error (without scrolling): 
![image](https://user-images.githubusercontent.com/9964343/77790565-139c5900-708b-11ea-8378-35281fd7bcaf.png)

Reply:
I get the issue... I don't understand how it happens. Ignore this thread and carry on. I'll investigate. Something is funky here.